compiler/circle-mpqsolver/src/bisection/BisectionSolver.cpp

```diff
@@ -261,6 +274,9 @@ std::unique_ptr<luci::Module> BisectionSolver::run(const std::string &module_pat
     }
   }
 
+  _hooks->on_final_MPQ(best_params, "uint8");
+  _hooks->on_end_iterations();
```

Comment:
Seems redundant. We may merge them into one (maybe on_end_iterations).

Reply:
> Seems redundant. We may merge them into one (maybe on_end_iterations).

Ok.